test/unit/reporters/MarkdownReporter-test.js

```diff
@@ -80,7 +80,7 @@ describe('MarkdownReporter', () => {
 
     it('should write the title to the buffer', done =>
       emitter.emit('start', '', () => {
-        assert.isOk(~mdReporter.buf.indexOf('Dredd'));
+        assert.isOk(mdReporter.buf.includes('Dredd'));
```

Comment:
I don't really know chai's assert that well, but should this be `assert.isTrue`?